sdk/loadtestservice/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -3,3 +3,10 @@
 ## 1.0.0b1 (2022-07-28)
 
 - Initial version
+
+## 1.0.0b2 (2022-08-04)
+
+- Added more test cases 
+- Added more samples 
+- Updated Readme
+- Fixed issues in "delete_app_component"
```

Comment:
please classify the content with https://azure.github.io/azure-sdk/policies_releases.html#change-logs